specs/core/0_beacon-chain.md

```diff
@@ -878,61 +855,37 @@ def get_seed(state: BeaconState, epoch: Epoch, domain_type: DomainType) -> Hash:
     return hash(domain_type + int_to_bytes(epoch, length=8) + mix)
 ```
 
-#### `get_committee_count`
+#### `get_committee_count_at_slot`
```

Comment:
This function is not in the ToC

Reply:
ah, thanks. need to re-run ToC generator